datavalidation.go

```diff
@@ -262,16 +262,20 @@ func (f *File) AddDataValidation(sheet string, dv *DataValidation) error {
 }
 
 // DeleteDataValidation delete data validation by given worksheet name and
```

Comment:
I suggest make sentence more clear, and format doc block with each line length limit. For example:

```text
// DeleteDataValidation delete data validation by given worksheet name and
// reference sequence. All data validations in the worksheet will be deleted
// if not specify reference sequence parameter.
```